File: app/src/main/java/mb00/android/codehub/ui/main/inject/MainActivityComponent.java
package mb00.android.codehub.ui.main.inject;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import mb00.android.codehub.ui.main.view.MainActivity;


@Subcomponent(modules = MainActivityModule.class)
public interface MainActivityComponent extends AndroidInjector<MainActivity> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<MainActivity> {}

}

File: app/src/main/java/mb00/android/codehub/ui/main/inject/MainActivityComponent.kt
package mb00.android.codehub.ui.main.inject

import dagger.Subcomponent
import dagger.android.AndroidInjector
import mb00.android.codehub.ui.main.view.MainActivity


@Subcomponent(modules = [
    MainActivityModule::class
])
interface MainActivityComponent : AndroidInjector<MainActivity> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<MainActivity>()

}
